designs/charlie/src/front.mjs

```diff
@@ -99,6 +96,45 @@ function draftCharlieFront({
   points.flyCurveCp1 = points.flyBottom.shiftFractionTowards(points.flyCorner, options.flyCurve)
   points.flyCurveCp2 = points.flyCurveStart.shiftFractionTowards(points.flyCorner, options.flyCurve)
 
+  let topStitchDist = (1 - options.flyWidth) * 8
+
+  points.flyTopSeamline = points.flyTop.shiftTowards(points.styleWaistIn, topStitchDist)
+
+  points.flyBottomSeamLine = utils.curveIntersectsY(
+    points.crotchSeamCurveStart,
+    points.crotchSeamCurveCp2,
+    points.crotchSeamCurveCp1,
+    points.fork,
+    points.flyBottom.shiftTowards(points.crotchSeamCurveStart, topStitchDist).y
+  )
+
+  paths.flyFacingLine = new Path()
+    .move(points.flyTop)
+    .line(points.flyCurveStart)
+    .curve(points.flyCurveCp2, points.flyCurveCp1, points.flyBottom)
+    .setClass('lining dashed')
+
+  let JseamCurve = paths.flyFacingLine.offset(-topStitchDist)
```

Comment:
Unless you plan to overwrite the variable later on, we prefer to use `const` in our codebase.
It's more declarative, somewhat more efficient, and reduces cognitive overhead when reading code.

Unless I missed something (I'm reading on my phone) this could be a `const`